--- a/src/burner/libretro/libretro.cpp	2022-03-22 20:57:35.535638300 +0800
+++ b/src/burner/libretro/libretro.cpp	2022-03-27 16:10:27.610067000 +0800
@@ -292,8 +292,8 @@
 
 	sprintf(library_version, "v%x.%x.%x.%02x %s", nBurnVer >> 20, (nBurnVer >> 16) & 0x0F, (nBurnVer >> 8) & 0xFF, nBurnVer & 0xFF, GIT_VERSION);
 
-	info->library_name = "HBFBNeo";
-	info->library_version = "严禁将本核心编译后使用在收费或者挂有募捐名义的项目中！！！";
+	info->library_name = "hbfbneo-专为街机改版游戏而生";
+	info->library_version = "严禁将本核心挂在任何EMUELEC整合包内!!!";
 	info->need_fullpath = true;
 	info->block_extract = true;
 	info->valid_extensions = "zip|7z|cue|ccd";
